web/wp-content/themes/borealis-2021/inc/partials/blog/blog-cite.php

```diff
@@ -0,0 +1,50 @@
+<?php 
+if (!function_exists('pg_generate_blog_sidebar_cite')) {
+    function pg_generate_blog_sidebar_cite($id) {
+        pg_svg_spritemap(); 
```

Comment:
for the spritemap you only need to include in the header file and you should be able to access it everywhere.